examples/foomedical/tsconfig.json

```diff
@@ -1,4 +1,5 @@
 {
+  "extends": "../../tsconfig.json",
```

Comment:
Sorry, can't do this, because `foomedical`, and all packages in `examples` get copied to separate repos.  These packages cannot reference anything outside of their respective folders.

Reply:
Fixed